api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java

```diff
@@ -25,7 +28,9 @@
 import org.openmrs.module.BaseModuleActivator;
 import org.openmrs.module.Extension;
 import org.openmrs.module.Module;
+import org.openmrs.module.ModuleUtil;

```

Comment:
Do you still need all these imports?

Reply:
@dkayiwa done

Reply:
But the ticket says you are still doing more work on this.